### DISCUSSION TITLE
The imageIPFS field in the networks.json file represents the icon of the network or the icon of the token of the paid service charge

### DISCUSSION THREAD

**Discussion by lucas7788** [snapshot-labs/snapshot#2318] [Q&A]
The imageIPFS field in the networks.json file represents the icon of the network or the icon of the token of the paid service charge?

**Comment by ChaituVR** [comment]
Hey @lucas7788 you can use https://www.pinata.cloud/ or some other pinning service, most of these services offer free storage
